docker/data/keycloak/realm-export.json

```diff
@@ -0,0 +1,2192 @@
+{
```

Comment:
Just put this file in `./keycloak` folder. I don't see any reasons to support such nested folder structure